assets/js/theme/account.js

```diff
@@ -104,7 +104,19 @@ export default class Account extends PageManager {
                         color: 'green',
                     },
                 },
-                countries: this.context.countries,
+                initializeData: {
```

Comment:
as a suggestion:  since we have a list of fields instead of `countries`, won't it be better to extract all needed fields from context and then add them as object shorthand props? Smth like this:

```
const { countries, paymentsUrl, storeHash, storeLocale } = this.context;
initializeData: { countries, paymentsUrl, storeHash, storeLocale }
```